### PR TITLE
feat: add storybook for Label component

### PR DESCRIPTION
**Pull request by Trjindal**
## What does this PR do?

Adds Label component to storybook

Fixes # (issue)
#2894 

https://github.com/user-attachments/assets/163f3e5d-3a6c-49ea-9689-664934136673

## How should this be tested?

-Run Storybook using` pnpm storybook`
- Inspect Storybook on `http://localhost:6006/`
### Required

- [x] Filled out the "How to test" section in this PR
- [x] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues
- [x] First PR at Formbricks? [Please sign the CLA!](https://cla-assistant.io/formbricks/formbricks) Without it we wont be able to merge it 🙏

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [x] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: packages/ui/Label/stories.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// src/Label.stories.tsx
+import type { Meta, StoryObj } from "@storybook/react";
+import { Label } from "./index";
+
+const meta: Meta<typeof Label> = {
```
Comment (gupta-piyush19) [inline packages/ui/Label/stories.tsx:5]: Please use the consistent approach to type things; other component's stories use `satisfies` to type the meta. Can you please make it the same here?

File: packages/ui/Label/stories.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  title: "UI/Label",
```
Comment (gupta-piyush19) [inline packages/ui/Label/stories.tsx:6]: set the title as `ui/Label`, again consistency 😅

File: packages/ui/Label/stories.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  component: Label,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "centered",
+    docs: {
+      description: {
+        component: `
+The **Label** component is used to label form fields in your application.
```
Comment (gupta-piyush19) [inline packages/ui/Label/stories.tsx:14]: The **Label** component is used to label the form fields.

Can you please update the description? Also, please add a description in other existing stories as well. 😊

**Review by gupta-piyush19 — CHANGES_REQUESTED** [review]
Thanks, @Trjindal, for the storybook feature. 😊 
change looks good and works great 🚀 
I've added a few small comments. Can you please fix them? Please fix them in your other PRs as well.
Thanks 🙌

File: packages/ui/Label/stories.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// src/Label.stories.tsx
```
[the remark this answers is not in the document]
Comment (Trjindal) [reply packages/ui/Label/stories.tsx]: Hi Piyush all the requested changes are done . please take a look. 😊

**Review by gupta-piyush19 — APPROVED** [review]
Thanks @Trjindal for the changes 😊 
LGTM 🚀

**Pull request merged**